core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -293,15 +293,7 @@ class PluginTypeChecker(platform: String = "onprem") extends Logging {
   }
 
   def isExprSupported(expr: String): Boolean = {
-    // Remove _ from the string. Example: collect_list => collectlist.
-    // collect_list is alias for CollectList aggregate function.
-    // An exception is date_format since the Expression and sql function name is different
-    // Expression: DateFormatClass, sql function- date_format
-    val exprLowercase = if (expr.equalsIgnoreCase("date_format")) {
-      expr
-    } else {
-      expr.toLowerCase.replace("_", "")
-    }
+    val exprLowercase = expr.toLowerCase
```

Comment:
I think what confused me that I did not realize that `readSupportedOperators` flattens the CV file and making each separate sql_function in its own row.

Reply:
That's correct, there is a separate row for each sql_function and Expression name.